pkg/deploy/devconfig.go

```diff
@@ -94,7 +94,9 @@ func DevConfig(_env env.Core) (*Config, error) {
 				SubscriptionID:      _env.SubscriptionID(),
 				RPResourceGroupName: os.Getenv("USER") + "-aro-" + _env.Location(),
 				Configuration: &Configuration{
+					AzureCloudName:       &_env.Environment().ActualCloudName,
 					DatabaseAccountName:  to.StringPtr(os.Getenv("USER") + "-aro-" + _env.Location()),
+					KeyvaultDNSSuffix:    &_env.Environment().KeyVaultDNSSuffix,
```

Comment:
```suggestion
					KeyvaultDNSSuffix:    to.StrPtr(&_env.Environment().KeyVaultDNSSuffix),
```

Reply:
`KeyVaultDNSSuffix` is a string in azure autorest vendored `environments.go`.

Reply:
using `&` is fine, I'm fine either way. `to.StrPtr` is a little more consistent